script/upgrades/cCOP/Config.sol

```diff
@@ -58,13 +59,15 @@ library cCOPConfig {
   function cCOPcUSD_PoolConfig(
     Contracts.Cache storage contracts
   ) internal view returns (Config.Pool memory poolConfig) {
-    // TODO: Get the final pool parameters from Roman and update them below.
+    // TODO: Get confirmation from Roman that these are OK
+    // These were taken from the $PUSO pool and adjusted to the COP/USD exchange rate,
+    // which was 0.00023747 at the time of writing
     poolConfig = Config.Pool({
       asset0: contracts.celoRegistry("StableToken"),
       asset1: contracts.deployed("StableTokenCOPProxy"),
       isConstantSum: true,
       spread: FixidityLib.newFixedFraction(3, 1000), // 0.3%, in line with current DT of chainlink feed
```

Comment:
@rcroessmann This was copied from $PUSO but I'm not sure if it's also the case for the COP/USD chainlink aggregator. I believe this is an off chain parameters so maybe we should check with them?

Reply:
confirm that it's also 0.3% https://data.chain.link/feeds/celo/mainnet/cop-usd